src/System.Reflection.Metadata/src/System/Reflection/Blob.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
Well, I need these files to be the same as in Roslyn. So that I can maintain them in both trees for a while and not go insane.